src/docs/output-targets/custom-elements.md

```diff
@@ -69,22 +69,11 @@ export const config: Config = {
-
-<!-- TODO(STENCIL-457): Move this info to the appropriate option on `customElementsExportBehavior` -->
-
-### autoDefineCustomElements
-
-By default, consumers of the `dist-custom-elements` output target need to
-register each Stencil component that they want to use in the bundle manually.
-Setting this flag to `true` will define a component automatically with the
-`CustomElementRegistry` when the component's module is imported. Note that this
-will also automatically recursively define any child components as well.
-
-This flag defaults to `false` when omitted from a Stencil configuration file.
+| Option                        | Description                                                                                                                                                                                                                                                                                                                                                                                               |
+| ----------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `default`                     | No additional re-export or auto-definition behavior will be performed.<br><br>This value will be used if no explicit value is set in the config, or if a given value is not a valid option.                                                                                                                                                                                                               |
+| `single-export-module`        | All component and custom element definition helper functions will be exported from the `index.js` file in the output directory (see [Defining Exported Custom Elements](#defining-exported-custom-elements) for more information on this file's purpose). This file can be used as the root module when distributing your component library, see [below](#distributing-custom-elements) for more details. |
+| `auto-define-custom-elements` | A component and its children will be automatically defined with the `CustomElementRegistry` when the component's module is imported.                                                                                                                                                                                                                                                                      |
```

Comment:
Does it make sense for us to order this alphabetically? I'm kind of on the fence here, since "default" is kind of a special value and I don't know if I'd prefer to differentiate it here by having it at the top. I suppose that doesn't prevent us from doing something like
- default
- auto-define-custom-elements
- single-export-module

At the same time, there's really nothing wrong with
- auto-define-custom-elements
- default
- single-export-module

either. Thoughts?

Reply:
I like keeping `default` at the top, but alphabetizing the others makes sense.